app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -82,8 +100,10 @@
     "Ireland",
     "Israel",
     "Italy",
+    "Ivory Coast",
```

Comment:
Ivory Coast doesn't appear to be listed on https://en.wikipedia.org/wiki/ISO_3166-1 ?